scene/2d/camera_2d.cpp

```diff
@@ -545,6 +592,26 @@ void Camera2D::_update_process_internal_for_smoothing() {
 	set_process_internal(enable);
 }
 
+void Camera2D::_set_limit_rect(const Rect2 &p_limit_rect) {
+	Point2 limit_rect_end = p_limit_rect.get_end();
+	set_limit(SIDE_LEFT, p_limit_rect.position.x);
+	set_limit(SIDE_TOP, p_limit_rect.position.y);
+	set_limit(SIDE_RIGHT, limit_rect_end.x);
+	set_limit(SIDE_BOTTOM, limit_rect_end.y);
+}
+
+Rect2 Camera2D::_get_limit_rect() const {
+	return Rect2(limit[SIDE_LEFT], limit[SIDE_TOP], limit[SIDE_RIGHT] - limit[SIDE_LEFT], limit[SIDE_BOTTOM] - limit[SIDE_TOP]);
+}
+
+void Camera2D::_set_limit_rect_to_viewport() {
+	Point2 origin = get_global_position();
+	set_limit(SIDE_LEFT, origin.x);
+	set_limit(SIDE_TOP, origin.y);
+	set_limit(SIDE_RIGHT, origin.x + float(GLOBAL_GET("display/window/size/viewport_width")));
+	set_limit(SIDE_BOTTOM, origin.y + float(GLOBAL_GET("display/window/size/viewport_height")));
+}
```

Comment:
I see you've removed the tool button from the code, but there's still this method left, now unused. So things are a little messed up now, I suspect you either forgot to remove everything button-related (including this method), or maybe you forgot to commit adding an EditorInspectorPlugin handling such button (suggested as an alternative in https://github.com/godotengine/godot/pull/101427#discussion_r1913834000)? :thinking:

Also PR description is not updated so it's hard to tell what's the intention now. Thus I've only skimmed through the code, haven't reviewed in detail.

Reply:
4.4 is feature frozen, no new features will be merged.